float64 x_min
float64 x_max
float64 y_min
float64 y_max
float64 z_min
float64 z_max
int8 selected
---
geometry_msgs/Transform transform_cable
bool result
